frontend/app/src/components/defi/Deposits.vue

```diff
@@ -240,17 +248,17 @@ onMounted(async () => {
         :profit="totalEarnedInAave"
       />
     </template>
-    <v-row class="loans__history mt-8" no-gutters>
-      <v-col cols="12">
-        <premium-card v-if="!premium" :title="tc('lending.history')" />
-        <lending-history
-          v-else
-          :loading="historyRefreshing"
-          :history="history"
-          :floating-precision="floatingPrecision"
-          @open-link="openUrl($event)"
-        />
-      </v-col>
-    </v-row>
+    <div v-if="!premium" class="mt-8">
+      <premium-card :title="tc('lending.history')" />
+    </div>
+    <div v-else>
+      <transaction-content
+        use-external-account-filter
+        :section-title="tc('common.events')"
+        :protocols="transactionEventProtocols"
+        :event-types="[HistoryEventType.DEPOSIT]"
```

Comment:
This table should not only show `DEPOSIT` events. We need to check what events we were showing before and replicate the event filter here.

Reply:
@yabirgb can you verify this, what events that are missing?

Reply:
@yabirgb can you provide a list of the events per lending protocol, including DSR.

Maybe we can also pre-add support for the missing events.